### PR TITLE
Leangseu edx/intl provider

### PR DESCRIPTION
**Pull request by leangseu-edx**
**TL;DR -** Paragon v20+ require usage of intl provider from react-intl v25+

JIRA: https://2u-internal.atlassian.net/browse/AU-871

**What changed?**

- Add `IntlProvider` for react components.
- https://github.com/openedx/paragon/releases/tag/v20.0.0

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/openedx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/openedx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by muselesscreator — APPROVED** [review]
LGTM

**Review by nsprenkle — APPROVED** [review]
Visual review looks good
